loader_hub/zulip/requirements.txt

```diff
@@ -0,0 +1,2 @@
+zulip
+llama_index
```

Comment:
don't need llama-index as a requirement (since the client to use llamahub is llama_index)